### PR TITLE
Misc stuff

### PR DESCRIPTION
**Pull request by ghost**
;_;

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.1'
+        classpath 'com.android.tools.build:gradle:2.3.3'
```
Comment (ghost) [inline build.gradle:8]: oh yeah and update dependency

**Comment by frap129** [comment]
"sphlash.xml" is spelled wrong, extra h. Other than that, looks good

**Comment by ghost** [comment]
was supposed to be a intentional
the flash == tha phlash ;-;
okay i dont make sense will fix that

**Comment by frap129** [comment]
Also, what is the point of switching the splash from a layout?

**Comment by ghost** [comment]
https://github.com/frap129/spectrum/pull/7/commits/518ca928fe5a1153db5dff92e285d3dbf9b16ea8
read the disc

**Pull request merged**